src/app/pages/apps/cost-centers/cost-center.component.html

```diff
@@ -0,0 +1,85 @@
+<vex-secondary-toolbar current="Edit Cost Center">
```

Comment:
We don't have add and edit cost center. Maybe remove it but do backup first so that we can revert if they want.